test/test_conversion_driver_esm1p5.py

```diff
@@ -142,3 +165,71 @@ def test_convert_esm1p5_output_dir_error():
         esm1p5_convert.convert_esm1p5_output_dir(
             "/test_convert_esm1p5_output_dir_error/fake/path/"
         )
+
+
+def test_format_successes():
+    succeeded_inputs = [
+        Path("dir_1/fake_file_1"),
+        Path("./dir_2/fake_file_2"),
+        Path("/dir_3/fake_file_3")
+    ]
+    succeeded_outputs = [
+        Path("./fake_output_dir/file1.nc"),
+        Path("fake_dir_2/output_file.nc"),
+        Path("/dir_500/ncfile.nc")
+    ]
+
+    succeeded_pairs = list(zip(succeeded_inputs, succeeded_outputs))
+    # Perhaps don't need to convert to list as format_successes just needs 
+    # an iterable as input?
+
+    success_reports = esm1p5_convert.format_successes(succeeded_pairs)
+
+    # Check that the successful outputs make it into the report
+    for i, successful_output in enumerate(succeeded_outputs):
+        assert str(successful_output) in success_reports[i]
+
+
+def test_format_failures_quiet():
+    failed = [
+        (Path("fake_file_1"), Exception("Error 1")),
+        (Path("fake_file_2"), Exception("Error 2")),
+        (Path("fake_file_3"), Exception("Error 3"))
+    ]
+
+    formatted_failure_reports = esm1p5_convert.format_failures(
+        failed,
+        True
+    )
+    for i, (file, exception) in enumerate(failed):
+        assert str(file) in formatted_failure_reports[i]
+        assert repr(exception) in formatted_failure_reports[i]
+
+
+
+def test_format_failures_not_quiet():
```

Comment:
Minor style thing: how does `test_format_failures_standard_mode()` sound?

Reply:
Yeah that sounds good! Just added in

Reply:
Why does it seem easier to think of var/func names when reviewing someone else's work? 🤔